### ISSUE TITLE
write performance of cam rh files

### ISSUE THREAD

**Issue by jedwards4b** [ESCOMP/CAM#1042]
### Issue Type

Infrastructure Update

### Issue Description

Restart history write performance is dreadful due to the ordering of variables in the 
file.  

### Will this change answers?

No

### Will you be implementing this yourself?

Yes

**Comment by jedwards4b** [comment]
To summarize the current state of this issue - A problem was found with PR #1043 and it was pulled from the merge.  However github marked it as merged and closed the PR (but not this issue.)   I am going to submit a new PR based on the latest camdev.

**Comment by cacraigucar** [comment]
I agree with what @jedwards4b said.  I double checked that the changes did not go into PR #1043 and they are not in the code.  Don't know why github did this, but I agree that opening an new PR appears to be the only way to proceed.

**Comment by cacraigucar** [comment]
This was brought in in #1131
